src/Lesson1/Task1/ArrayExcerciseVariantOne.java

```diff
@@ -0,0 +1,83 @@
+package Lesson1.Task1;
+
+/*
+* 1 Написать программу, осуществляющую обработку массива. Исходные условия:
+массив содержит только целые числа от -10 до 10 (допускается генерация
+элементов с помощью метода (int) Math.random()), программа должна выводить
+в консоль исходный массив и полученный результат, количество элементов в
+массиве 20.
+Варианты задания:
+1. В массиве целых чисел поменять местами максимальный отрицательный
+элемент и минимальный положительный.*/
```

Comment:
Please, think what will be with you task if generated array will have only positive or only negative values.